deployments/k8s/helm/signalfx-agent/templates/configmap.yaml

```diff
@@ -10,8 +10,14 @@ metadata:
     chart: {{ template "signalfx-agent.chart" . }}
 data:
   agent.yaml: |
-{{- if .Values.agentConfig }}
-{{ toYaml .Values.agentConfig | indent 4 }}
+{{ if .Values.agentConfig }}
+  {{- $agentConf := .Values.agentConfig }}
+  {{- $confType := printf "%T" $agentConf }}
+  {{- if eq $confType "string" }}
```

Comment:
There’s a typeOf function but otherwise it’s what I had in mind.

Reply:
Yeah let's use something more explicit than the `%T` flag on printf if it's available.  Will make this easier to understand for anybody looking back at it.